env/plugin-commands-env/src/node.ts

```diff
@@ -41,6 +41,7 @@ const nodeFetchPromises: Record<string, Promise<string>> = {}
 
 export async function prepareExecutionEnv (config: NvmNodeCommandOptions, { extraBinPaths, executionEnv }: PrepareExecutionEnvOptions): Promise<PrepareExecutionEnvResult> {
   if (!executionEnv?.nodeVersion) return { extraBinPaths: extraBinPaths ?? [] }
+  else if (executionEnv.nodeVersion === process.versions.node) return { extraBinPaths: extraBinPaths ?? [] }
```

Comment:
Could we combine these?

```suggestion
  if (!executionEnv?.nodeVersion || executionEnv.nodeVersion === process.versions.node) {
    return { extraBinPaths: extraBinPaths ?? [] }
  }
```

Reply:
That's actually a good point. APplied :+1: Thanks

Reply:
Thanks!